src/main/java/frc/robot/subsystems/Storage.java

```diff
@@ -107,6 +106,18 @@ public void setTopOutput(double output) {
     mTopRoller.set(ControlMode.PercentOutput, output * SPEED_FACTOR);
   }
 
+  public void increaseBallCount() {
```

Comment:
Overall, I think we are likely safe from race conditions, but incr/decr commands are the ones that should likely be synchronized in a singleton pattern

Reply:
Done